application/backend/src/main/resources/apps/typerefinery/components/widgets/tabcontainer/.content.json

```diff
@@ -1,5 +1,5 @@
 {
   "group": "Typerefinery - Widgets",
   "sling:resourceType": "ws:Component",
-  "title": "Tab"
+  "title": "Tab Container"
```

Comment:
can we just call this Tabs, this component its not a container as per say as you can't drop anything into it.

Reply:
Yep. done